strings/base_version.h

```diff
@@ -3,7 +3,7 @@
 
 // WINRT_version is used by Microsoft to analyze C++/WinRT library adoption and inform future product decisions.
 extern "C"
-__declspec(selectany)
+inline __declspec(selectany)
```

Comment:
Why is this necessary?

Reply:
Because it is in a struct

Reply:
The compiler is implicitly in-lining this anyway.

Reply:
Then leave it alone. Unless it's causing compilation issues, this has worked as-is for a few years and I don't want to disrupt that unnecessarily.